internal/xds/translator/translator.go

```diff
@@ -195,7 +195,10 @@ func (t *Translator) processHTTPListenerXdsTranslation(tCtx *types.ResourceVersi
 			}
 
 			// 1:1 between IR HTTPRoute and xDS config.route.v3.Route
-			xdsRoute := buildXdsRoute(httpRoute)
+			xdsRoute, err := buildXdsRoute(httpRoute)
```

Comment:
translation is best effort, we cannot bail out early and break other routes

Reply:
I think the order here doesn't matter because DeltaXDS will eventually make the order correct when pushing all the resources to Envoy.

Reply:
we cannot err out due to syntactic issues or field validation issues, it must be due to a system issue

Reply:
I had this doubt, but still did this way because all the other code in aslo returns eraly.

Should I change all of them? Or maybe we address this issue in a follow-up PR?

Reply:
sure, can you raise a GH issue for this

Reply:
https://github.com/envoyproxy/gateway/issues/2200